### DISCUSSION TITLE
AllTalk TTS Generator polish special characters question.

### DISCUSSION THREAD

**Discussion by Borostwory** [erew123/alltalk_tts#398] [General]
Hello, I noticed the AllTalk TTS Generator strips the text input from polish special characters.
When using Demo/Test TTS all is fine.
Is this a bug or am I doing something wrong?
![Untitled-1](https://github.com/user-attachments/assets/cafb550a-3a80-4a47-9994-af345a3c36d8)

**Comment by erew123** [comment]
@Borostwory Please use AllTalk V2 https://github.com/erew123/alltalk_tts/tree/alltalkbeta?tab=readme-ov-file#alltalk-tts-v2-beta

![image](https://github.com/user-attachments/assets/d070befb-87be-4aaa-a9a6-6e2b29ec43b6)

![image](https://github.com/user-attachments/assets/c486dead-6d9a-4809-b881-496297dc29b1)

Thanks

**Reply by Borostwory** [reply]
Thank you!
Works perfectly.
